lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -346,19 +347,27 @@ def sendDeactDedicatedBearerAccept(self, ue_id, bearerId):
         )
         print("************* Sending deactivate EPS bearer context accept\n")
 
-    def sendPdnConnectivityReq(self, ue_id, apn):
+    def sendPdnConnectivityReq(
+        self, ue_id, apn, pdn_type=1, pcscf_addr_type=None
+    ):
         req = s1ap_types.uepdnConReq_t()
         req.ue_Id = ue_id
         # Initial Request
         req.reqType = 1
         req.pdnType_pr.pres = 1
-        # PDN Type = IPv4
-        req.pdnType_pr.pdn_type = 1
+        # PDN Type 1 = IPv4, 2 = IPv6, 3 = IPv4v6
+        req.pdnType_pr.pdn_type = pdn_type
         req.pdnAPN_pr.pres = 1
         req.pdnAPN_pr.len = len(apn)
         req.pdnAPN_pr.pdn_apn = (ctypes.c_ubyte * 100)(
             *[ctypes.c_ubyte(ord(c)) for c in apn[:100]]
         )
+        print("********* PDN type", pdn_type)
```

Comment:
I made the same comment in the other diff, so probably i am wrong :D, but why don't you add an space or a dash?

`print("********* PDN type-", pdn_type)`

Reply:
Space is automatically added. This is how the print looks - 
********* PDN type 1
********* pcscf_addr_type ipv4